### PR TITLE
Add the enabled flag.

### PR DESCRIPTION
**Pull request by Natim**
r? @magopian

### CODE REVIEW

File: xml2kinto/id_generator.py [hunk 1]
```diff
@@ -10,6 +10,9 @@ def create_id(data):
     This will be used to make sure that two deeply nested data structures are
     exactly equal.
     """
+    # We should ignore the enabled key.
+    if 'enabled' in data:
+        del data['enabled']
```
Comment (magopian) [inline xml2kinto/id_generator.py]: isn't this mutating the passed in data? We want to keep this data, right? Just not use it for the id computation? (btw, why don't we want it or the id computation?)

Comment (Natim) [inline xml2kinto/id_generator.py]: we should definitely do a deepcopy there.

File: xml2kinto/tests/test_id_generator.py [hunk 2]
```diff
@@ -11,3 +11,23 @@ def test_create_id():
             "children": ["foo", "bar", "foobar"]
         }
     }) == 'ef636b94-97a2-3c5e-5a64-22e23cf28e8f'
+
+
+def test_create_id_ignores_enabled_flag():
+    """It should handle strings, list, int and dict."""
```
Comment (magopian) [inline xml2kinto/tests/test_id_generator.py]: This docstring is a bit misleading, does it mean it should "use" everything but booleans? Why?

File: xml2kinto/synchronize.py [hunk 3]
```diff
@@ -34,6 +34,8 @@ def push_changes(diff, kinto_client, bucket, collection):
         for record in to_delete:
             batch.delete_record(record)
         for record in to_create:
+            # Add the enabled flag.
```
Comment (magopian) [inline xml2kinto/synchronize.py]: maybe `Records are enabled by default`?

File: xml2kinto/tests/test_id_generator.py (hunk 2 above)
Comment (Natim) [inline xml2kinto/tests/test_id_generator.py]: No it should handle everything.

**Comment by Natim** [comment]
Rebased

**Pull request merged**